source/patterns/03-templates/page-grid/page-grid.twig

```diff
@@ -28,9 +28,8 @@
       {%- with arguments|merge({_template: template}) only -%}
 
         {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['page-grid__item', 'page-grid__main']), role: 'main'}) -%}
-        {%- set element = "main" -%}
 
-        {%- include _template -%}
+        {%- include _template with {element: 'main'} -%}
```

Comment:
This is also https://github.com/friendsoftwig/twigcs/issues/35, but I prefer the change.